docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -30,35 +30,26 @@ Strapi prepopulates the `./config/middlewares.js` file with built-in, internal m
 
 The `./config/middlewares.js` file exports an array, where order matters and controls the execution order of the middleware stack:
 
-<!-- ? What is "enabled: we could make enabling dynamic" for (see comments in code)? These were in the RFC. Could we use `enabled: false` just like we can disable plugins in ./config/plugins.js? What's the status on this (just an idea, decided but not implemented, etc.)? -->
```

Comment:
it's enabled if it's in the array / disable if it's not. We don't have stuff auto enabled in the background so we don't need this trick to be able to disable things just removing them from the array disables them. It's a lot more simple and transparent.